docs/cubeviz/displaycubes.rst

```diff
@@ -20,12 +20,12 @@ converted to lower case):
     - Mask viewer: ``hdu.data.dtype`` is `int`, `numpy.uint` or `numpy.uint32`, or
       ``hdu.name`` is in the set ``['mask', 'dq']``
 
-If any viewer fails to populate automatically, or if displaying 
-different data is desired, the user can manually select data for each viewer
-as described in the next section. Different statistics for collapsing the 
-spectrum displayed in the spectrum viewer can be chosen as described in 
+The next section describes how to manually select data in cases where a viewer
+fails to populate automatically or a user wants to change the data displayed.
```

Comment:
"fails to populate" (also in the original wording) makes this sounds like a bug... is there an underlying bug this is referring to?  Or does this just mean cases where the imported data doesn't contain the mask and/or uncertainty information?